tests/integration/tier0/latest-kernel-check/test_latest_kernel_check.py

```diff
@@ -5,13 +5,12 @@
 
 
 @pytest.mark.failed_repoquery
-def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2rhel):
+def test_verify_latest_kernel_check_passes_with_failed_repoquery(get_system_release, shell, convert2rhel):
     """
     This test verifies, that failed repoquery is handled correctly.
     Failed repoquery subsequently caused the latest kernel check to fail.
     Introduced fixes should get the process past the latest kernel check.
     """
-    get_system_release = platform.platform()
```

Comment:
Since it's used many times during the test, I would store it in a variable and used the variable during the test.
something like
```
system_release = get_system_release()
```

Reply:
This was removed, now we are using the get_system_release from the conftest.py

Reply:
Yay, I just realized that we cannot call conftest function on the global level here.